src/pages/ServiceList/ItemDescription.tsx

```diff
@@ -51,22 +54,25 @@ export default class ItemDescription extends React.PureComponent<Props, State> {
   render() {
     return this.state.health ? (
       <Row>
-        <Col xs={12} sm={12} md={4} lg={4}>
+        <Col xs={12} sm={12} md={this.state.columnWidth} lg={this.state.columnWidth}>
           <strong>Health: </strong>
           <HealthIndicator id={this.props.item.name} health={this.state.health} mode={DisplayMode.SMALL} />
         </Col>
-        <Col xs={12} sm={12} md={4} lg={4}>
+        <Col xs={12} sm={12} md={this.state.columnWidth} lg={this.state.columnWidth}>
           {!this.props.item.istioSidecar && <MissingSidecar />}
         </Col>
-        <Col xs={12} sm={12} md={4} lg={4}>
+        <Col xs={12} sm={12} md={this.state.columnWidth} lg={this.state.columnWidth}>
           <strong>Config: </strong>{' '}
           <ConfigIndicator
             id={this.props.item.name + '-config-validation'}
             validations={[this.props.item.validation]}
             size="medium"
           />
         </Col>
-        <Col xs={12} sm={12} md={4} lg={4} />
+        <Col xs={12} sm={12} md={this.state.columnWidth} lg={this.state.columnWidth}>
```

Comment:
Does it help about the width if the code just hide the whole `Col` if it's  not present?

Reply:
Actually there's already an empty column that doesn't show, but if something is in there,  then the total width is 16 and it breaks the columns.
To avoid to change the design for people not using the API functionality, I changed the columns to 3 3 3 3 instead of 4 4 4 (empty one being ignored). I'm opened to suggestions